sources/engine/Stride.Rendering/Rendering/Images/Outline/OutlineEffect.sdsl

```diff
@@ -0,0 +1,69 @@
+namespace Stride.Rendering.Images
+{
+    /// <summary>
+    /// Simple fog
+    /// </summary>
+    internal shader OutlineEffect : ImageEffectShader
+    {
+        stage float2 ScreenDiffs;  // .x = Width, .y = Height
+
+        stage float zFar;
+        stage float zNear;
+
+        stage float NormalWeight;
+        stage float DepthWeight;
+        stage float NormalNearCutoff;
+
+        stage Texture2D DepthTexture;
+        
+        float3 normal_from_depth(float depth, float2 texcoords) {
```

Comment:
Mix of inline and next line braces

Reply:
Resolving this, because it's the same as in the main branch.